Formula/convox.rb

```diff
@@ -0,0 +1,15 @@
+class Convox < Formula
+  desc "The convox AWS PaaS CLI tool"
+  homepage "https://convox.com/"
+  url "https://bin.equinox.io/a/9RUvivxx6Lm/convox-20160606232025-darwin-amd64.tar.gz"
```

Comment:
It looks like they support the `vendor` system, which would make life a bit easier. Simple case of changing the URL to https://github.com/convox/rack/archive/20160615213630.tar.gz, adjusting the SHA256 & then something similar to [this](https://github.com/Homebrew/homebrew-core/blob/fb81645562f7e495a49561065912788af2d1f14f/Formula/artifactory-cli-go.rb#L14-L20).